.changeset/cuddly-eels-lay.md

```diff
@@ -0,0 +1,5 @@
+---
+"chainlink": minor
+---
+
+Improve TXM performance by optimize Confirmer and Finalizer queries to stop pulling EVM receipt. #internal #changed #updated
```

Comment:
I think you can tag this as just `#internal` since the change/update doesn't affect anything externally.